scripts/release/docs_update_experiments.sh

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bash
+
+# Usage: ./docs_update_experiments.sh
+#
+# This script updates the available experimental features in the documentation.
+# It fetches the latest mainline and stable releases to extract the available
+# experiments and their descriptions. The script will update the
+# feature-stages.md file with a table of the latest experimental features.
+
+set -euo pipefail
+# shellcheck source=scripts/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")/../lib.sh"
+cdroot
+
+# From install.sh
+echo_latest_stable_version() {
+	# https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c#gistcomment-2758860
+	version="$(curl -fsSLI -o /dev/null -w "%{url_effective}" https://github.com/coder/coder/releases/latest)"
+	version="${version#https://github.com/coder/coder/releases/tag/v}"
+	echo "v${version}"
+}
+
+echo_latest_mainline_version() {
+	# Fetch the releases from the GitHub API, sort by version number,
+	# and take the first result. Note that we're sorting by space-
+	# separated numbers and without utilizing the sort -V flag for the
+	# best compatibility.
+	echo "v$(
+		curl -fsSL https://api.github.com/repos/coder/coder/releases |
```

Comment:
curious: can we add a small go app to mimic `jq`? I'd like to prevent such command trains

Reply:
It could be simplified for sure. This comment intended to adopt any go-`jq` solution.